firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -247,6 +251,19 @@ public void updateToNewRelease_whenNewAabReleaseAvailable_showsUpdateDialog() {
         shadowOf(dialog).getMessage().toString());
   }
 
+  @Test
+  public void updateToNewRelease_fromABackgroundThread_showsUpdateDialog()
```

Comment:
(Nit/ No action required) I'm just now noticing that some of these tests use the old name of `updateToNewRelease`, and some use the new name `updateIfNewReleaseAvailable` X)

Reply:
Ah, good catch. Fixed!